generators/server/templates/gradle/testcontainers.gradle.ejs

```diff
@@ -0,0 +1,53 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+/*
+* Run <code>./gradlew integrationTest -Ptestcontainers</code> to run integration tests
+* with testcontainers.
+* See https://www.testcontainers.org/modules/databases/ for more details
+<%_ if (prodDatabaseType === 'mssql') { _%>
+* You are required to accept EULA for sql server containers.
+* Follow the instructions at https://www.testcontainers.org/modules/databases/mssqlserver/
+<%_ } _%>
+<%_ if (prodDatabaseType === 'oracle') { _%>
+* You need to specifying a docker image name via config.
+* Follow the instructions at https://www.testcontainers.org/modules/databases/oraclexe/
+<%_ } _%>
+*/
+dependencies {
+    testImplementation "org.testcontainers:testcontainers"
+    <%_ if (prodDatabaseType === 'mysql') { _%>
+    testImplementation "org.testcontainers:mysql"
+    <%_ } _%>
+    <%_ if (prodDatabaseType === 'mariadb') { _%>
+    testImplementation "org.testcontainers:mariadb"
+    <%_ } _%>
+    <%_ if (prodDatabaseType === 'postgresql') { _%>
+    testImplementation "org.testcontainers:postgresql"
+    <%_ } _%>
+    <%_ if (prodDatabaseType === 'mssql') { _%>
+    testImplementation "org.testcontainers:mssqlserver"
+    <%_ } _%>
+    <%_ if (prodDatabaseType === 'oracle') { _%>
+    testImplementation "org.testcontainers:oracle-xe"
+    <%_ } _%>
+}
+
+if (project.hasProperty('testcontainers')) {
```

Comment:
I think you forgot to apply this gradle skript (see https://github.com/intesys/generator-jhipster-testcontainers/blob/master/generators/app/index.js#L95). But maybe having this in a dedicated gradle script is a bit too much?
If we want to keep it separate I would suggest to check for the `testcontainers` property in the main file and apply this script only when the property is present

Reply:
Makes sense. In the module wanted something separate. I'll try to bring everything into the build.gradle.